hack/check-charts.sh

```diff
@@ -29,7 +29,7 @@ if [[ -d "$1" ]]; then
   echo "Checking whether all charts can be rendered"
   for chart_dir in $(find charts -type d -exec test -f '{}'/Chart.yaml \; -print -prune | sort); do
     [ -f "$chart_dir/values-test.yaml" ] && values_files="-f $chart_dir/values-test.yaml" || unset values_files
-    helm template $values_files "$chart_dir" 3>&1 1>/dev/null 2>&3 | (grep -v "found symbolic link in path" || true)
+    helm template $values_files "$chart_dir" > /dev/null 2> >(sed '/found symbolic link in path/d' >&2)
```

Comment:
Is `$values_files` not double quoted intentionally? If yes that means we expect to receive multiple whitespaced positional parameters? Looked at `helm template --help` to see how positional parameters were handled but seems to me it can't handle arbitrary number of positional parameters
```
helm template --help
...
helm template [NAME] [CHART] [flags]
...

Reply:
It is intentional.
As you can check values file is being set `values_files="-f $chart_dir/values-test.yaml"` so if we use "values_files" it will run helm command as below -
```
helm template '-f ./charts/gardener/controlplane/values-test.yaml` charts/gardener/controlplane
```
here is `-f some_value` is considered as on input value which is not what we want.

Reply:
@Kostov6 PTAL.